e2e_tests/standard/test_multiple.py

```diff
@@ -68,6 +68,9 @@ def test_single_thread(
 
         assert len(container[namespace].fetch()) == number_of_reinitialized + 1
 
+        for r in reinitialized:
```

Comment:
Why?

Reply:
To cleanup after test (otherwise spawned processes stay alive outside of scope of this test). And why not? ;)